### DISCUSSION TITLE
[Feature Request] Support Material You themed icons on Android 13

### DISCUSSION THREAD

**Discussion by Basaingeal** [PaulWoitaschek/Voice#1458] [Archive]
![Screenshot_20220821-113218.png](https://user-images.githubusercontent.com/36519219/185801474-69fcea91-ef9a-4a38-9a54-6f7ab8ff1e45.png)

**Comment by PaulWoitaschek** [comment]
Implemented as of https://github.com/PaulWoitaschek/Voice/commit/15f3cfd657de0d83f4ee3eed79bc41ee513fadef

![Screenshot_20220826_161452](https://user-images.githubusercontent.com/743462/186923889-9b00a7ac-94e1-49c6-b89b-5e6f92cefc7c.png)
![Screenshot_20220826_161517](https://user-images.githubusercontent.com/743462/186923855-3c783713-ecfa-4589-ad0b-ea4352ee5bdc.png)
